store/mongo/tenant_datastore.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package mongo
+
+import (
+	"context"
+)
+
+type TenantStoreMongo struct {
+	db *DataStoreMongo
+}
+
+func NewTenantStoreMongo(db *DataStoreMongo) *TenantStoreMongo {
+	return &TenantStoreMongo{
+		db: db,
+	}
+}
+
+func (ts *TenantStoreMongo) MigrateTenant(ctx context.Context, id string) error {
+	return ts.db.MigrateTenant(ctx, DbVersion, id)
```

Comment:
why not just expose DbVersion to the outside world and use directly in App?

Reply:
It is unfortunately and it's defied in `store/mongo` I believe. It's also used in the `main` package and in datastore tests where `store/mongo` is imported. `App` on the other hand is in `useradm` package which does not import `store/mongo` and only works with dependencies that were passed from main.Importing `store/mongo` there would feel a bit off.